department-of-veterans-affairs/caseflow
# changed text on confirm cancellation modal

Resolves [APPEALS-67759](https://jira.devops.va.gov/browse/APPEALS-67759)

# Description
the Confirmation Cancellation and Reassign model now displays the text: "Cancellation of task is final. "

## Acceptance Criteria

- [ ] When the Confirmation Cancellation and Reassign model is displayed the text reads: Cancellation of task is final. 

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below

1.Run seed data, bundle exec rake db:seed:legacy_appeals_for_vlj_movement_testing
2. Enable feature toggle, FeatureToggle.enable!(:legacy_case_movement_scm_to_vlj_for_blockhtask)
3. Switch to User "TEAM_ADMIN"
4. Go to their queue, switch views to the Board Of Veteran's Appeals Cases
5. Filter by Legacy Appeal to find the specific seeds with their Veteran Name
6. Open all appeals, then verify no task action is available on the active blocking hearing task.
7. With the appeals open, in another tab switch users to "BVARDUNKLE" who is an SCM User.
8. Refresh the appeal case details pages
9. Verify that there is a Task Action dropdown next to the HearingTask that allows the SCM User to Cancel Blocking Tasks and Advance To Judge
10. Click the Task Action and Advance through the workflow all the way to the Confirm Cancellation Modal.
11. Confirm text "Cancellation of task is final. " is displayed in the modal
